internal/common/grpc/gateway.go

```diff
@@ -8,12 +8,10 @@ import (
 	"strings"
 
 	"github.com/go-openapi/runtime/middleware"
-	gwruntime "github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/jcmturner/gokrb5/v8/spnego"
 	"google.golang.org/grpc"
 
-	"github.com/grpc-ecosystem/grpc-gateway/runtime"
-
 	protoutil "github.com/G-Research/armada/internal/armada/protoutils"
```

Comment:
Not for this PR, but for some nebulous list of minor cleanups, is it really that useful to have `protoutil` instead of `protoutils`?

Reply:
Should probably be fixed too